CSharp/VirtoCommerce.MemberExModule.Template/Data/Models/SupplierDataEntity.cs

```diff
@@ -42,39 +44,39 @@ public override MemberDataEntity FromModel(Member member, PrimaryKeyResolvingMap
                 }
             }
 
-            // Here you can write code for custom mapping
-            // supplier properties will be mapped in base method implementation by using value injection
             return retVal;
         }
+        
         /// <summary>
         /// This method used to convert data type instance to domain model
         /// </summary>
         public override Member ToModel(Member member)
         {
-            // Here you can write code for custom mapping
-            // supplier properties will be mapped in base method implementation by using value injection
+            // Here you can write code for custom mapping.
+            // Member properties will be mapped in base method implementation by using value injection
             var retVal = base.ToModel(member) as Supplier;
 
             if (retVal != null)
                 retVal.Reviews = Reviews.OrderBy(x => x.Id).Select(x => x.ToModel(AbstractTypeFactory<SupplierReview>.TryCreateInstance())).ToList();
```

Comment:
why OrderBy(..) is used? Any reason to sort?

Reply:
Yes you are right. Fixed.